### DISCUSSION TITLE
How to add two inverters?

### DISCUSSION THREAD

**Discussion by Ackesan2** [djansen1987/SAJeSolar#59] [Q&A]
Hi community,

One question, how can I add two inverters with one single request?

BR Acke

**Comment by deweirdt** [comment]
Hi, I'm also seeking for a solution like this. I believe today the 'Plant' is being read. Not the details of the inverters.
I'm having the same problem. 1 Plant with 2 Inverters.

**Comment by djansen1987** [comment]
It is currently not possible because the config wil create the same entity names for both (you could add the config twice in your configuration.yaml) there are some preparations for it in the code but where never finished. I am happy to merge a PR if someone wants to create the code but do not have time my self to add it any time soon. Sorry :(
